### ISSUE TITLE
Add `SourceLocation` to `ModuleSym`

### ISSUE THREAD

**Issue by magnus-madsen** [flix/flix#5968]
... that spans the entire module...

**Comment by mlutze** [comment]
I don't think it is difficult. But it will be limited since even something like `mod A/B/C` is read as a declaration of `A`, and of `B`, and of `C`. So multi-declarations will be pervasive.

**Comment by magnus-madsen** [comment]
Uh oh. Hmm.

Still, it would actually allow us to test the quickfix part for single namespace modules.
